components/src/organisms/DeckLabelSet/index.tsx

```diff
@@ -46,7 +46,7 @@ const DeckLabelSetComponent = (
           ? deckLabels.map((deckLabel, index) => (
               <DeckLabel
                 key={`DeckLabel_${index}`}
-                maxWidth={`calc(${width}px - ${SPACING.spacing16})`}
+                maxWidth={`calc(${width}px - ${SPACING.spacing8})`}
```

Comment:
nit
we avoid using SPACING to width/height.

```suggestion
                maxWidth={`calc(${width}px - 8px)`}
```